### DISCUSSION TITLE
How do I remove "Powered by Supabase" watermark in the email footer?

### DISCUSSION THREAD

**Discussion by luminrs** [supabase/supabase#20680] [Questions] [answered]
Hello everybody,

I'm thrilled that I discovered Supabase, it's been a delight to build with. Currently, I'm moving a dev environment into production, and I've set custom STMP for my email (I'm using the Supabase managed solution by the way). It seems that the email is showing "Powered by Supabase" in the footer of the email, I don't want to confuse my users when they see this, many of them won't know what Supabase is and it may look suspicious to the average user. I attempted to hide it using CSS however since the email isn't that long it will just show it in the preview before you click the email in Gmail or iCloud, however it _is_ hidden when you click the email. 

Anyway I don't want to have to do some work around to hide it, is there any way to remove this? Is there any official support for removing the watermark in the paid plans? Any advice is much appreciated.

Wishing everybody the best,
luminrs

**Comment by GaryAustin1** [comment, marked as answer]
Pro plans allow/do remove it.
![image](https://github.com/supabase/supabase/assets/54564956/c8e7d68c-ffc1-41ab-b7d3-af59daabfaa1)

**Reply by luminrs** [reply]
Should have looked more thoroughly. Thanks Gary.
